### PR TITLE
Add intrinsics _mm_div_pd

### PR DESCRIPTION
**Pull request by JishinMaster**

### CODE REVIEW

**Review by jserv — CHANGES_REQUESTED** [review]
Please rebase the latest "master" branch.

**Comment by JishinMaster** [comment]
What is the proper way to rebase to the master?
Since the FMA pull request is still pending, I updated my master branch with a git merge before creating this feature branch.
Hence the work in progress commits which appear but are not directly related.
I am no git expert :(

**Comment by marktwtn** [comment]
> What is the proper way to rebase to the master?
> Since the FMA pull request is still pending, I updated my master branch with a git merge before creating this feature branch.
> Hence the work in progress commits which appear but are not directly related.
> I am no git expert :(

- Get the newest master/main branch
- Checkout to the branch you are working with
- `$ git rebase master`
-  Fix conflict and make sure the git rebase success
-  `$ git rebase -i master` and make the multiple commits into single one, the make sure the git rebase success

File: sse2neon.h [hunk 1]
```diff
@@ -3433,6 +3433,27 @@ FORCE_INLINE __m128 _mm_div_ss(__m128 a, __m128 b)
         vsetq_lane_f32(value, vreinterpretq_f32_m128(a), 0));
 }
 
+// Divides the two double precision, floating-point values of a and b.
+//
+//   r0 := a0 / b0
+//   r1 := a1 / b1
+//
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_div_pd
```
Comment (marktwtn) [inline sse2neon.h:3441]: Please use the description and operation of [_mm_div_pd](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_div_pd)

**Review by marktwtn — CHANGES_REQUESTED** [review]

**Review by marktwtn — APPROVED** [review]

**Review by marktwtn — APPROVED** [review]

**Comment by jserv** [comment]
Thank @JishinMaster for contributing!

**Pull request merged**